config/sidebar.yml

```diff
@@ -49,6 +49,9 @@ articles:
       - title: "Dynamic Client Registration"
         url: "/api-auth/dynamic-client-registration"
 
+      - title: "Token Introspection"
+        url: "/api-auth/token-introspection"
+
```

Comment:
Can you also add an entry for this doc at the api-auth index? https://auth0.com/docs/api-auth
Same place, after `Dynamic Client Registration`.